canvas_admin_tools/urls.py

```diff
@@ -4,10 +4,12 @@
 
 
 urlpatterns = [
+    url(r'^django-rq/', include('django_rq.urls')),
```

Comment:
I removed the django-rq urls because you are right we won't be able to access them due to LTI authentication.